osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -51,10 +51,14 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
 
             if (score.Mods.Any(h => h is OsuModRelax))
             {
+                // https://www.desmos.com/calculator/adhhgjtuyp
+                double okMultiplier = osuAttributes.OverallDifficulty > 0.0 ? 1 - Math.Pow(osuAttributes.OverallDifficulty / 12.0, 2) : 1.0;
+                double mehMultiplier = osuAttributes.OverallDifficulty > 0.0 ? 1 - Math.Pow(osuAttributes.OverallDifficulty / 12.0, 6) : 1.0;
+
                 // As we're adding Oks and Mehs to an approximated number of combo breaks the result can be higher than total hits in specific scenarios (which breaks some calculations) so we need to clamp it.
-                effectiveMissCount = Math.Min(effectiveMissCount + countOk + countMeh, totalHits);
+                effectiveMissCount = Math.Min(effectiveMissCount + countOk * okMultiplier + countMeh * mehMultiplier, totalHits);
```

Comment:
OD can actually be higher than 12 with 2.0x @ OD11, [which means we can get negative `effectiveMissCount`.](https://i.imgur.com/gCLIENr.png) Max OD should be used here, though I'm not sure what that is off the top of my head.

Reply:
![image](https://user-images.githubusercontent.com/8269193/177205936-418f0b5a-424b-4855-8a62-9f8e06ca1d4d.png)
Fixed, OD 13.33 was chosen because its an OD at which point great hitwindow becomes 0